src/arlo_e2e/ray_helpers.py

```diff
@@ -26,6 +26,7 @@ def ray_init_localhost(
         ray.init(num_cpus=num_cpus if num_cpus > 0 else os.cpu_count())
         _ray_is_local = True
         ray_post_init(write_failure_probability)
+        ray_wait_for_workers()
```

Comment:
I actually threw that in there because I thought it was going to fix a spurious bug I was seeing. I'm flipping it to the same order as the remote case.